config/initializers/008-rack-cors.rb

```diff
@@ -25,15 +25,19 @@ def call(env)
     status, headers, body = @app.call(env)
     headers ||= {}
 
-    Discourse::Cors.apply_headers(cors_origins, env, headers) if cors_origins
+    Discourse::Cors.apply_headers(cors_origins, env, headers)
 
     [status, headers, body]
   end
 
   def self.apply_headers(cors_origins, env, headers)
-    origin = nil
+    if env['SCRIPT_NAME'] == "/assets" && ['GET', 'OPTIONS'].include?(env['REQUEST_METHOD']) && Discourse.is_cdn_request?(env)
+      headers['Access-Control-Allow-Origin'] = '*'
+      headers['Access-Control-Allow-Credentials'] = 'false'
```

Comment:
As above

> If you don't need credentials, omit this header entirely (rather than setting its value to false).

Also... why are we doing this in two places? Could we do it all in the middleware, rather in both rails and middleware?